build/images/Dockerfile.build.windows

```diff
@@ -52,5 +52,7 @@ COPY --from=windows-ovs /Windows/System32/vcruntime140.dll /Windows/System32/
 COPY --from=windows-ovs /Windows/System32/libeay32.dll /Windows/System32/
 COPY --from=windows-ovs /Windows/System32/ssleay32.dll /Windows/System32/
 
+COPY build/images/scripts/* /
```

Comment:
Can we follow the way it is done in L43 for this?

Reply:
We are not copying the scripts from any other image here . Since we are copying from antrea repo itself. You can refer to this :
https://github.com/antrea-io/antrea/blob/9d9fa73afc8383b929d9843773bd2b93b794c323/build/images/Dockerfile.build.ubuntu#L41

Reply:
image `antrea-build-windows` will also contain this script.